src/Api/Client/AccountsClient.php

```diff
@@ -241,4 +241,23 @@ public function healthCheck()
 
         return $this->getClient()->get();
     }
+
+    /**
+     * @var string $backOfficeUrl
+     * @var string $frontendUrl
+     * @var int $multiShopId
+     *
+     * @return array
+     */
+    public function createOauth2Client($backOfficeUrl, $frontendUrl, $multiShopId) {
```

Comment:
```suggestion
    public function createShopIdentity($backOfficeUrl, $frontendUrl, $multiShopId) {
```
I guess Mr. Tournerie as many "to_dip_into_the_yoghurt" in its kitchen. Personally I use "spoons" :trollface: